modules/ROOT/pages/tokenization-logs.adoc

```diff
@@ -1,6 +1,6 @@
 == Tokenization Service Logs
 
-To determine traffic counts, use the tokenization summary message that is returned in Runtime Fabric logs. To retrieve the logs, configure log forwarding in Runtime Fabric. 
+To determine traffic counts, use the tokenization summary message that is returned in Runtime Fabric logs. To retrieve the logs, configure log forwarding in Runtime Fabric.
```

Comment:
Is the topic that follows supposed to be a description of how to configure log forwarding in Runtime Fabric?
Do that task included in the See Also link?

Reply:
I'm a little confused about why retrieving the logs precedes enabling the logs....

Reply:
This is only one topic, which covers logging for tokenization. RTF logs (for traffic counts) and audit logs (interactions of tokenization and detokenization). Two diff types of logs but still under the umbrella of tokenization service logs.